genie-web/src/main/java/com/netflix/genie/web/controllers/ApplicationRestController.java

```diff
@@ -135,7 +135,7 @@ public ApplicationRestController(
     @DeleteMapping
     @ResponseStatus(HttpStatus.NO_CONTENT)
     public void deleteAllApplications() throws GenieException {
-        log.debug("Delete all Applications");
+        log.warn("Called to delete all Applications");
```

Comment:
Not worth changing, but I think this should be INFO. It's nothing out of the extraordinary, it's just an entry point.
Same for the ones below.